init-mqtt.h

```diff
@@ -75,26 +85,20 @@ bool saveMqttConfig(const char* server, int port, const char* user, const char*
   json[MQTT_CONFIG_USER] = user;
   json[MQTT_CONFIG_PWD] = pwd;
 
-  res = saveJson(mqttFileName, json);
-
-  if (res) {
+  if (saveJson(mqttFileName, json)) {
```

Comment:
We can rewrite this line like:
if ((res = saveJson(...)) { } to avoid additional assignment(line 89).
Think if this is more readable.

Reply:
no
I want to have only 2 assignments of res
res = false when starting
and res = true in successfull scenario
that looks the most straightforward

Reply:
this conversation can be resolved
I'll let you know when other fixes are replied - so please leave all other conversations opened